libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -973,6 +973,57 @@
     <output name="out" type="vector3" default="0.0, 0.0, 0.0" />
   </nodedef>
 
+  <!--
+    Node: <unifiednoise2d>
+    Combines the available noises into an artist-friendly node, adding controls
+    to adjust frequency, offset, jitter, etc... Where possible controls have
+    been unified between noises; any noise-specific controls should live in a
+    dedicated folder. These folders should be hidden by artist tools, unless their
+    corresponding noise type is selected.
+    
+    The noise types are:
+    
+       0 - Noise (noise2d/noise3d)
```

Comment:
This is a minor note, but the documentation needs an update from "Noise" to "Perlin" to match the latest set of enums.